src/pybind/mgr/orchestrator/module.py

```diff
@@ -369,11 +369,21 @@ def _update_set_addr(self, hostname: str, addr: str) -> HandleCommandResult:
         return HandleCommandResult(stdout=completion.result_str())
 
     @_cli_read_command('orch host ls')
-    def _get_hosts(self, format: Format = Format.plain) -> HandleCommandResult:
+    def _get_hosts(self, format: Format = Format.plain, matches_name: str = '', label: str = '', host_status: str = '') -> HandleCommandResult:
```

Comment:
```suggestion
    def _get_hosts(self, format: Format = Format.plain, host_pattern: str = '', label: str = '', host_status: str = '') -> HandleCommandResult:
```

Similar to 

```yaml
placement
  host_pattern: 'storage-1[3-7]'
```
?

Reply:
+1 for consistency. Although, TBH both matches_name and host_patterm feel overly verbose.